scripts/module/ramp.nut

```diff
@@ -24,22 +28,31 @@ function onPlayerGameKeysChange( player, oldKey, newKey ) {
       angle2 = Vector(0, 0, angle);
     }
     pos = GetForwardPoint( pos, angle, 10.0 );
-    local obj = CreateObject( 562, player.World, pos, 220 );
-    obj.RotateToEuler(angle2, 0);
-    RampObj[player.ID] = obj;
+    local publicRampObj = CreateObject( 562, player.World, pos, 177 );
```

Comment:
I commented on the other issue about 177 being possibly too high. Ideally I'd like to see a screenshot of this and a lower value.

Reply:
I'll attach that screenshot tonight!